src/Tester/Tester.csproj

```diff
@@ -110,6 +115,10 @@
       <Link>OrleansConfigurationForTesting.xml</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Content Include="..\OrleansTestingHost\OrleansDependencyResolverConfigurationForTesting.xml">
```

Comment:
This `OrleansDependencyResolverConfigurationForTesting.xml` config file should be in the Tester project, and not in the OrleansTestingHost project.